src/frontend/src/lib/utils/nav.utils.ts

```diff
@@ -30,14 +36,14 @@ const tokenUrl = ({
 	path
 }: {
 	token: Token;
-	path: '/transactions/' | '/';
+	path: AppPath.Transactions | undefined;
```

Comment:
It couldn't be `undefined` before, the previous description was incorrect?

Reply:
I removed the leading '/' from the input, and moved it inside the function here

``${path ?? ''}/?${TOKEN_PARAM}=...`

so it seemed more correct to have `undefined` input instead of empty string

Reply:
When the user hits `https://oisy.com` the path will be undefined?

Reply:
correct!

for example

`https://oisy.com/?token=ETH`

will have undefined path